driver-core/src/main/com/mongodb/connection/Cluster.java

```diff
@@ -45,6 +45,13 @@
      */
     ClusterDescription getDescription();
 
+    /**
+     * Get the current description of this cluster.
+     *
+     * @return the current ClusterDescription representing the current state of the cluster or null
```

Comment:
This shouldn't be allowed to return null and in practice it does not.  Both subclasses of BaseCluster ensure that it's initialized in their respective constructors

Reply:
Done.